tests/multicomponent_compressible_projected_density_box.prm

```diff
@@ -0,0 +1,151 @@
+# This is a testcase that demonstrates the projected density approximation within 
+# The multicomponent compressible material model.
+# The setup is a 1x1 m, 2x2 grid where we apply a surface pressure equal 
+# to 50 GPa with gravity disabled. This creates a uniform pressure field that 
+# results in a uniform density distribution. 
+# This allows for no material to flow into/out of the model while allowing 
```

Comment:
What about the model allows material to flow in and out? The boundary conditions have not yet been described above.